### PR TITLE
Added stepper

### PR DESCRIPTION
**Pull request by itaistopel**
image

### CODE REVIEW

**Comment by ofirassif** [comment]
Its not really clear that that is what the purple bar does..

**Comment by svechinsky** [comment]
Have a look at how it's done in the figma screens, might make sense to split the questions into af ew components. 
Or just have all the questions in one component and have the bar move from screen to screen.
https://www.figma.com/file/bnnHW8Lx4HCTpURSz2YJ9z/%D7%90%D7%A4?node-id=0%3A1

**Comment by itaistopel** [comment]
> Have a look at how it's done in the figma screens, might make sense to split the questions into af ew components.
> Or just have all the questions in one component and have the bar move from screen to screen.
> https://www.figma.com/file/bnnHW8Lx4HCTpURSz2YJ9z/%D7%90%D7%A4?node-id=0%3A1

You talk about this? 
image

What about a stepper for the entire app?

**Comment by svechinsky** [comment]
You can just use the same concept for the entire app.
Step 1 - questons
Step 2 - camera
Step 3 - mic

**Comment by itaistopel** [comment]
image

@svechinsky @SaparovNiv

File: app/src/main/java/com/example/coronadiagnosticapp/ui/activities/MainActivity.kt [hunk 1]
```diff
@@ -26,6 +27,12 @@ private val REQUIRED_PERMISSIONS = arrayOf(
 class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
+        val res: Resources = this.resources
+        val dm: DisplayMetrics = res.displayMetrics
+        val conf: Configuration = res.configuration
+//        conf.setLocale(Locale("iw")) // API 17+ only.
```
Comment (ofirassif) [inline app/src/main/java/com/example/coronadiagnosticapp/ui/activities/MainActivity.kt:33]: ?

File: app/src/main/java/com/example/coronadiagnosticapp/ui/fragments/resultFragment/ResultFragment.kt [hunk 2]
```diff
@@ -31,6 +31,7 @@ class ResultFragment : ScopedFragment() {
         activity?.applicationContext.let { ctx ->
             (ctx as MyApplication).getAppComponent().inject(this)
         }
+//        activity?.findViewById<Stepper>(R.id.Stepper)?.forward()
```
Comment (ofirassif) [inline app/src/main/java/com/example/coronadiagnosticapp/ui/fragments/resultFragment/ResultFragment.kt]: ?

File: app/src/main/java/com/example/coronadiagnosticapp/ui/activities/MainActivity.kt (hunk 1 above)
Comment (itaistopel) [reply app/src/main/java/com/example/coronadiagnosticapp/ui/activities/MainActivity.kt:33]: I added this so we will be able to easily switch between the languages of the app

**Review by svechinsky — APPROVED** [review]

**Pull request merged**